docusaurus/docs/reference/metadata.md

```diff
@@ -308,3 +309,15 @@ Parameters for the token authentication request.
 | `grant_type`    | string | No       | OAuth grant type                                                                          |
 | `resource`      | string | No       | OAuth resource                                                                            |
 
+## extensions
+
+The list of extensions that the plugin registers under other extension points.
```

Comment:
"below" instead of "under"

Reply:
Thanks! Fixed in https://github.com/grafana/plugin-tools/pull/1027/commits/d86a27882baf20783c722f78383ef9ee0965a309.